### PR TITLE
opensearch-dashboards 2.0.0

### PR DESCRIPTION
**Pull request by cho-m**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by SMillerDev — APPROVED** [review]

File: Formula/opensearch-dashboards.rb [hunk 1]
```diff
@@ -4,19 +4,20 @@ class OpensearchDashboards < Formula
...
   license "Apache-2.0"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "849889b4333f56fdca11b839fa6c7dfbebe70b6cca4dafdb624f1d2c4154b2e6"
   end
 
   depends_on "yarn" => :build
-  depends_on "node@10" # Switch to `node` after https://github.com/opensearch-project/OpenSearch-Dashboards/issues/406
+  depends_on arch: :x86_64
+  depends_on "node@14"
```
Comment (bayandin) [inline Formula/opensearch-dashboards.rb:17]: We can still mention the same issue for switching to node 16/18.

File: Formula/opensearch-dashboards.rb [hunk 2]
```diff
@@ -4,19 +4,20 @@ class OpensearchDashboards < Formula
...
+      revision: "3d6dd638d021f383a4c6ab750c83a1d30d3787b3"
   license "Apache-2.0"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "849889b4333f56fdca11b839fa6c7dfbebe70b6cca4dafdb624f1d2c4154b2e6"
   end
 
   depends_on "yarn" => :build
-  depends_on "node@10" # Switch to `node` after https://github.com/opensearch-project/OpenSearch-Dashboards/issues/406
+  depends_on arch: :x86_64
```
Comment (bayandin) [inline Formula/opensearch-dashboards.rb:16]: Is there an upstream issue for arm support?

File: Formula/opensearch-dashboards.rb (hunk 1 above)
Comment (cho-m) [reply Formula/opensearch-dashboards.rb:17]: Added comments in #102427

File: Formula/opensearch-dashboards.rb (hunk 2 above)
Comment (cho-m) [reply Formula/opensearch-dashboards.rb:16]: I didn't see any so I created one https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1630.

**Pull request closed without merging**